releases/v1.1.0.toml

```diff
@@ -0,0 +1,150 @@
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.0"
+
+pre_release = true
+
+preface = """\
+`containerd` provides a daemon for managing running containers.
+
+1.1 is the second major release for `containerd` with added support for CRI, the
+Kubernetes [Container Runtime Interface](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md).
+CRI is a new plugin which allows connecting the containerd daemon directly to a
+Kubernetes kubelet to be used as the container runtime. The CRI GRPC interface
+listens on the same socket as the containerd GRPC interface and runs in the same
+process.
+
+In addition to all of the stability and bug fixes backported to 1.0,
+1.1 includes...
+
+- CRI plugin
+- ZFS and AUFS snapshotter
+- Improvements to the `ctr` tool
+- Better support for multiple platforms
+- Cross namespace content sharing
+- Better mount cleanup
+- Support for disabling plugins
+- TCP debug address for remote debugging
+- Update to Go 1.10
+- Improvements to the garbage collector
+
+## CRI Plugin
+
+This release of `cri` is a native plugin of `containerd`. It is built into
+`containerd` v1.1 and CRI services are enabled by default.
+
+You can now use Kubernetes, with `containerd` directly, without having to use
+the intermediate `cri-containerd` daemon. The `cri-containerd` daemon is
+end-of-life.
+
+*Note: Please [drain your node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) before
+upgrading from older versions of `cri-containerd` to `containerd` v1.1.*
+
+You can [use a containerd config file to configure the `cri` plugin](https://github.com/containerd/cri/blob/v1.0/docs/config.md).
+
+### Untrusted Workload Runtime
+
+To run an untrusted pod on a runtime for untrusted workload, such as
+[Kata Containers](https://katacontainers.io/) or
+[Clear Containers](https://clearlinux.org/containers), you can:
+1. Configure a runtime for untrusted workload [with the config option `plugins.cri.containerd.untrusted_workload_runtime`](https://github.com/containerd/cri/blob/v1.0/docs/config.md).
+2. Create an untrusted pod by setting the annotation `io.kubernetes.cri.untrusted-workload` to `"true"`, for example:
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  annotations:
+    io.kubernetes.cri.untrusted-workload: "true"
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```
+
+By default, `cri` will run pods with the default runtime. However, if a pod has
+the `io.kubernetes.cri.untrusted-workload` annotation, the `cri` plugin will run
+the pod with the runtime for untrusted workloads.
+
+Unless configured otherwise, the default runtime is set to
+[runc](https://github.com/opencontainers/runc).
+
+### Container Runtime Interface v1alpha2
+
+The supported CRI (Container Runtime Interface) version for Kubernetes v1.10 is
+now `v1alpha2.` This release of `cri` has been updated to use CRI `v1alpha2`, so
```

Comment:
`cri` -> `containerd/cri`, or just "the CRI plugin"?

Reply:
@Random-Liu any recommendation? I think maybe the project name `containerd/cri` is what is intended here